harness/determined/pytorch/_pytorch_trial.py

```diff
@@ -897,9 +892,45 @@ def _train_batch(self, batch: pytorch.TorchData, epoch_idx: int, batch_idx: int)
                 f"train_batch() must return a dictionary mapping string names to Tensor metrics, "
                 f"got {type(training_metrics).__name__}"
             )
-
         return training_metrics
 
+    def _log_tb_metrics(
```

Comment:
oh!  Is there a way you can put this function in our `det.pytorch._metric_utils`?  We keep some of our shared pytorchtrial/deepspeed utils in there, and I think this function would fit in nicely there.  All you'd need to do is pass the tensorboard writer in, and this wouldn't even use `self`, right?